index.html

```diff
@@ -155,7 +155,7 @@ <h2>Data Model</h2>
             <h3>JsonSchema2023</h3>
             <p>
                This <a href="https://www.w3.org/TR/json-ld/#dfn-term-definition">term definition</a>
-               is <code>https://www.w3.org/TR/vc-json-schema/#JsonSchema2023</code>.
+               is <code>https://www.w3.org/TR/vc-json-schema#JsonSchema2023</code>.
```

Comment:
The slashless URI gets a `302 Found` redirect to the slashed URI, where the fragment does not resolve.

The text found in that document always reads as the mixed case `JsonSchema2023`, but the fragment identifier for the section where it is defined is the lowercase [`#jsonschema2023`](https://www.w3.org/TR/vc-json-schema/#jsonschema2023), where it says --
> This [term definition](https://www.w3.org/TR/json-ld/#dfn-term-definition) is https://www.w3.org/ns/credentials#JsonSchema2023.

-- which gets a `302 Found` redirect to `https://www.w3.org/ns/credentials/` which gets a `404 Not Found`. 

This is confusing to me, and I've been working with all this stuff for quite a while. It seems that anyone new to any of this material who encounters the above will quickly throw up their hands and walk away.